ansible/playbooks/utils/check_volume.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: Checking for necessary --extra_vars
+  hosts: all
+  vars:
+    example:
+      - "-e \"{'VOLUME_DEVICE':'/vol1', 'VOLUME_MKFS_DEVICE':True, 'VOLUME_DEVICE_TYPE': 'ext4'}\""
+
+  tasks:
+
+    - fail: msg="Must set VOLUME_DEVICE. Pass a value in via `-e` as JSON {{example[0]}}"
+      when: "{{ VOLUME_DEVICE is undefined }}"
```

Comment:
Does this code do what you intend? I think the double curly braces are for variable expansion, and I don't think `is undefined` means anything.

I think we want `when: VOLUME_DEVICE is not defined`